build.gradle

```diff
@@ -1,15 +1,17 @@
 buildscript {
     repositories {
         jcenter()
+        google()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.2'
```

Comment:
In the current version, we are using version 2.10, the same as Bitcoin Wallet, because that allows the NDK features to work for scrypt and X11 native hashing.